diff --git a/dist/vite/node/adapter.js b/dist/vite/node/adapter.js
index fea1dbd3ceb48b6bb27cc4277a8a24221282f1c3..7569af6398289e520b75315875dafa3b6a274c6d 100644
--- a/dist/vite/node/adapter.js
+++ b/dist/vite/node/adapter.js
@@ -46,7 +46,7 @@ function createRequest(req, res) {
   let init = {
     method: req.method,
     headers: createHeaders(req.headers),
-    signal: controller.signal
+    // signal: controller.signal
   };
   if (req.method !== "GET" && req.method !== "HEAD") {
     init.body = node.createReadableStreamFromReadable(req);
